Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs

```diff
@@ -129,6 +137,17 @@ protected override void ApplySettings()
                 FindObjectOfType<InputManager>().GetComponent<SimpleSinglePointerSelector>().Cursor = FindObjectOfType<Cursor>();
             }
 
+            if (Values[SceneSetting.AddSpatialMapping])
+            {
+                var spatialMappingManagers = FindObjectsOfType<SpatialMappingManager>();
+                foreach (var spatialMappingManager in spatialMappingManagers)
+                {
+                    DestroyImmediate(spatialMappingManager.gameObject.GetParentRoot());
```

Comment:
Do you want to destroy the GameObject?

What if there's other components ppl are using on it?

Reply:
destroying the parent seems risky.

should we just leave what we find alone if it

also, we should warn users if we find more than one pre-existing spatial mapping manager

Reply:
Ahh yeah, I copied the line above which destroys the cursor's parent: https://github.com/keveleigh/HoloToolkit-Unity/blob/9525e0803aacc26dfa13b493a0f44300c656137d/Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs#L132

I can update it to instead just destroy its own GameObject, like the InputSystem does (multiple times): https://github.com/keveleigh/HoloToolkit-Unity/blob/9525e0803aacc26dfa13b493a0f44300c656137d/Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs#L92

or should all of this behavior be updated?

Reply:
I'd rather have a warning of too many spatial mapping managers and leave the existing one(s) untouched